src/dmd/cli.d

```diff
@@ -369,8 +369,15 @@ dmd -cov -unittest myprog.d
         Option("Hf=<filename>",
             "write 'header' file to filename"
         ),
-        Option("HC",
-            "generate C++ 'header' file"
+        Option("HC=[silent|verbose]",
```

Comment:
In general, this DMD style is unintuitive to me. `[...]` would be something optional in my eyes. Something like `HC[=<silent|verbose>]` (I'm sure you get the meaning) would IMO be preferrable, keeping backwards compatibility, but allowing to use the new explicit variants.

Reply:
Good point, changed it to `-HC[=[silent|verbose]]` based on the help text for `-check`.